src/components/forms/text-field/RuiTextField.vue

```diff
@@ -123,9 +122,17 @@ const showClearIcon = logicAnd(
 );
 
 function clearIconClicked() {
-  emit('update:modelValue', '');
+  set(modelValue, '');
   emit('clear');
 }
+
+watchDebounced(
+  focused,
+  (focused) => {
+    set(focusedDebounced, focused);
+  },
+  { debounce: 500 },
+);
```

Comment:
can't we use [refDebounced](https://vueuse.org/shared/refDebounced/) from vueuse instead of a watcher here?

Reply:
hmm yeah I should use refDebounced instead